pysidt/sidt.py

```diff
@@ -318,7 +346,7 @@ def to_dict(obj):
     for attr in attrs:
         val = getattr(obj, attr)
 
-        if callable(val) or val == getattr(obj.__class__, attr):
```

Comment:
It seems like this is the change that's breaking the notebook. Why do we need it to be obj.__class__()?